functions/index.js

```diff
@@ -23,8 +22,12 @@ app.post('/api/requests', async (req, res) => {
       requestedAt: new Date().toString()
     }
     const requestCollection = db.collection(requestCollectionName);
-    const querySnapshot = await requestCollection.where("pantherID", "==", request.pantherID).where("course", "==", request.course ).get()
-
+    const querySnapshot = await requestCollection.where("pantherID", "==", request.pantherID).where("course", "==", request.course ).get();
```

Comment:
Scope issue here. If you declare the variable inside this try block, you can't access it inside the lower try block. You'll have to create a var before the two try blocks (set it equal to null) and modify it in the first one then you can access it in the second one.